src/students/students.service.ts

```diff
@@ -34,25 +35,12 @@ export class StudentsService {
     }
   }
 
-  async manageResponse(status: string, message: string, data?: any) {
-    // Following common standard for API response.
-    let messages: string[] = [];
-    let response: Iresponse = null;
-    messages.push(message);
-    response = {
-      status: status,
-      messages: messages,
-    };
-    return data ? { ...response, data: data } : response;
-  }
-
   async findAll() {
-    const data = await this.dbService.student.findMany();
-    return await this.manageResponse('success', 'unit test', data);
+    return await this.dbService.student.findMany();
   }
 
-  findOne(id: number) {
-    return this.dbService.student.findFirst({
+  async findOne(id: number) {
+    return await this.dbService.student.findFirst({
```

Comment:
Sorry, this code was written by me. I forgot to update. @Noman5237 as you are working in this module currently, please replace the Response structure according to AdminServices. from my last push.

Reply:
yes this was inconsistent, i will update it